src/index.html

```diff
@@ -1,14 +1,67 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home" class="logo">
+        <img src="./moyo_1.svg" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item" >
+            <a href="#" class="nav__link is-active">
+              <div class = "nav__text">Apple</div>
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">
+              <div class = "nav__text">Samsung</div>
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">
+              <div class = "nav__text">Smartphones</div>
+            </a>
+          </li>
+          <li class="nav__item" data-qa="hover">
+            <a href="#laptopsandcomputers" class="nav__link">
+              <div class = "nav__text"> Laptops & Computers </div>
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#gadgets"
+            class="nav__link">
+              <div class = "nav__text">Gadgets</div>
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">
+              <div class = "nav__text">Tablets</div>
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">
+              <div class = "nav__text">Photo</div>
+            </a>
+          </li>
+          <li class="nav__item">
+            <a href="#video" class="nav__link">
+              <div class = "nav__text">Video</div>
+            </a>
+          </li>
```

Comment:
please add the spaces between the li blocks as it will make a code cleaner

Reply:
Added